src/components/Surfaces/MessageBar/MessageBar.js

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { getStyles } from './MessageBar.styles';
+import { classNamesFunction, customizable, styled } from 'office-ui-fabric-react/lib/Utilities';
+import { MessageBar as FabricMessageBar } from 'office-ui-fabric-react/lib/MessageBar';
+
+export function MessageBar(props) {
+	const { getStyles, messageBarType, isMultiline, onDismiss } = props;
+	const classNames = classNamesFunction()(getStyles, props);
+	return (
+		<FabricMessageBar
+			className={classNames.root}
+			messageBarType={messageBarType}
+			isMultiline={isMultiline}
+			onDismiss={onDismiss}>
+			{props.children}
+		</FabricMessageBar>
+	);
+}
+
+MessageBar.propTypes = {
+	/** Message bar type */
+	messageBarType: PropTypes.any, //TODO enum?
```

Comment:
TODO comment prevents 'isMultiline' props description to be parsed. Remove it

Reply:
Use PropTypes.oneOf(['type 1', 'type 2', 'type 3' .....]) instead of 'any'. Enums are available in typescript only.